phidgets_api/include/phidgets_api/encoder.h

```diff
@@ -76,6 +76,12 @@ class Encoder final
      * @param enabled The new powered state of the encoder*/
     void setEnabled(bool enabled) const;
 
+    Phidget_EncoderIOMode getIOMode() const;
+    void setIOMode(Phidget_EncoderIOMode io_mode) const;
+
+    uint32_t getDataInterval() const;
+    void setDataInterval(uint32_t interval) const;
```

Comment:
Could you rename `interval` to `data_interval_ms` or `interval_ms` for consistency with the rest of the code base? (Unfortunately the code base isn't quite consistent here either).